zuul.d/ansible-security-jobs.yaml

```diff
@@ -151,3 +151,9 @@
     timeout: 3600
     vars:
       ansible_collections_repo: github.com/ansible-collections/trendmicro.deepsec
+
+- job:
```

Comment:
This is missing a nodeset.

The way to test this, would be to open a PR in trendmicro collection, then depends-on this PR. That would run your job